baseapp/baseapp/activity_log/permissions.py

```
@@ -0,0 +1,25 @@
+from django.contrib.auth.backends import BaseBackend
+
+
+class ActivityLogPermissionsBackend(BaseBackend):
+    def has_perm(self, user_obj, perm, obj=None):
+        if perm == "activity_log.list_activitylog_any_visibility":
+            return True if user_obj.is_superuser else False
```

Ruff

Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)